frontend/cypress/e2e/profile-page.cy.ts

```diff
@@ -21,20 +21,20 @@ describe("Profile page tests", () => {
   });
 
   it("Try to go to profile page without logging in", () => {
-    cy.visit(Cypress.env("baseUrl") + "/profile");
+    cy.visit(Cypress.env("baseUrl") + "/profile/settings");
     cy.url().should("eq", Cypress.env("baseUrl") + "/");
   });
 
   it("Go to profile page as logged in user", () => {
     cy.login(newUser.email, newUser.password);
-    cy.visit(Cypress.env("baseUrl") + "/profile");
+    cy.visit(Cypress.env("baseUrl") + "/profile/settings");
     cy.get(".section-header").contains("Settings");
-    cy.url().should("eq", Cypress.env("baseUrl") + "/profile");
+    cy.url().should("eq", Cypress.env("baseUrl") + "/profile/settings");
```

Comment:
Nämä `cy.url()` voisi korvata `cy.location("pathname").should("eq", "/profile/settings");`